tests/queries/0_stateless/01660_system_parts_smoke_long.sql

```diff
@@ -1,26 +1,37 @@
--- There is different code path when:
--- - _state is not requested
--- - _state is requested
--- - only _state is requested
+-- There is different code path for the following cases:
 SELECT * FROM system.parts FORMAT Null;
 SELECT *, _state FROM system.parts FORMAT Null;
 SELECT _state FROM system.parts FORMAT Null;
+SELECT *, _use_count FROM system.parts FORMAT Null;
```

Comment:
I'm ok if use_count will be selected with `*`.
BTW, if that will be true, let's also add to `system.parts_columns`